backend/ctgov/script/etl.py

```diff
@@ -0,0 +1,161 @@
+import psycopg2
+import re
+
+from datetime import datetime
+from dotenv import load_dotenv
+from os import getenv
+from os.path import abspath
+from psycopg2 import Error
+
+dotenv_path = abspath("../../core/.env")
```

Comment:
use join, dirname and __file__

Reply:
So here it would be 
`from os.path import join, dirname`

`dotenv_path = join(dirname(dirname(dirname(__file__))), ".env")